le-thesaurus.el

```diff
@@ -127,14 +133,19 @@ Synonyms are sorted by similarity."
                      (cons (region-beginning) (region-end))
                    (bounds-of-thing-at-point 'symbol)))
          (word (buffer-substring-no-properties (car bounds) (cdr bounds)))
+	 (word-case (le-thesaurus--get-word-case word))
          (results (le-thesaurus--ask-thesaurus-for-synonyms word))
          (completions (le-thesaurus--get-completions results))
          (replace-text (completing-read
                         (format "Select synonym for %S: " word)
                         (le-thesaurus--completing-read-collection-fn completions))))
     (when bounds
       (delete-region (car bounds) (cdr bounds))
-      (insert replace-text))))
+      (insert (if (equal word-case 'upcase)
```

Comment:
might as well use `cond` here as well :)

Reply:
Right! Don't know why I didn't do it.

Reply:
It's fixed now :)